nucleus/admin/config-api/src/main/java/org/glassfish/config/support/TranslatedConfigView.java

```diff
@@ -303,5 +297,19 @@ public static String getRealPasswordFromAlias(final String at) throws
         return new String(domainPasswordAliasStore.get(an));
     }
 
+    private static String expandPasswordAlias(String alias) {
```

Comment:
Can't see a caller for this method.

Reply:
Yep, left behind from a refactor. Thanks :)